### PR TITLE
Fix #179142 mrbeast.store

### LINKED ISSUE #179142: mrbeast.store
### Issue URL (Ads)
 [https://mrbeast.store/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmrbeast.store%2F)
Username: `@xglossyntx`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.46
System version: | Windows 10
Browser: | Firefox
Stealth mode options: | Block WebRTC, <br>Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Self-destructing third-party cookies (2880), <br>Block trackers
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><details><summary><b>Language-specific:</b></summary>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>AdGuard French, <br/>AdGuard Chinese, <br/>EasyList Czech and Slovak, <br/>EasyList Italy, <br/>AdBlockID, <br/>List-KR, <br/>Dandelion Sprout's Nordic Filters, <br/>IndianList</details>

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #179142

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Review by piquark6046 — DISMISSED** [review]
I think that we can generalize the rules to the following rule:
```adblock
/.well-known/shopify/monorail
```

It also exists in EasyPrivacy.

**Comment by xglossyntx** [comment]
I was going to do that if I saw one more site with this tracker, but I might as well just do it now and get it over with.

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -81,8 +82,8 @@
 ||stats.bio.link/api/event/pageView?
 ||shopify.com/__dux|
 ||shopify.com/__dux|$ping,redirect-rule=nooptext
-/shopify/monorail/*/produce$domain=runpacers.com|aquariumcoop.com|online.citysuper.com.hk|windsorstore.com|clothesmentor.com|fashionnova.com|lttstore.com
-/shopify/monorail/*/produce$ping,redirect-rule=nooptext,domain=runpacers.com|aquariumcoop.com|online.citysuper.com.hk|windsorstore.com|clothesmentor.com|fashionnova.com|lttstore.com
+/shopify/monorail/*/produce$domain=runpacers.com|aquariumcoop.com|online.citysuper.com.hk|windsorstore.com|clothesmentor.com|fashionnova.com|lttstore.com|mrbeast.store
```
Comment (piquark6046) [inline SpywareFilter/sections/specific.txt:85]: I think that we can generalize the rules to the following rule:
```adblock
/.well-known/shopify/monorail
```

It also exists in EasyPrivacy.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

**Comment by xglossyntx** [comment]
Is the current placement of the new rule okay, or would you like me to move it to the top of the list?

**Review by zloyden — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**